src/main/java/org/opentripplanner/routing/algorithm/filterchain/package.md

```diff
@@ -0,0 +1,36 @@
+# ItineraryListFilterChain
```

Comment:
May we should make a link to this documentation in the https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/ARCHITECTURE.md doc.